pyaction/PYACTION_NEXT_INSERT_KW.DATA

```diff
@@ -0,0 +1,571 @@
+-- 
+-- REGIONS SECTION 
+-- 
+-- =================================================================================================================================
+REGIONS
+--
+--       ARRAY       CONSTANT     ---------- BOX ---------                 
+--                                I1  I2   J1  J2   K1  K2                
+EQUALS                                                  
+         EQLNUM      1            1*  1*   1*  1*   1*  1* / SET EQLNUM FOR ALL CELLS         
+         FIPNUM      1            1   20   1   14   1   10 / SET FIPNUM 1 AREA
+         FIPNUM      2            1   20   15  30   1   10 / SET FIPNUM 2 AREA
+         SATNUM      1            1*  1*   1*  1*   1*  1* / SET SATNUM FOR ALL CELLS          
+/                                                                                   
+--                                                                              
+--       DEFINE THE FLUID IN-PLACE NAMES AND REGION AND NUMBERS                            
+-- 
+FIPABC
+         2000*1  2000*2  2000*3        /
+
+-- =================================================================================================================================
+-- 
+-- SOLUTION SECTION 
+-- 
+-- =================================================================================================================================
+SOLUTION  
+--
+--       DATUM   DATUM   OWC     PCOW   GOC    PCGO   RS   RV   N           
+--       DEPTH   PRESS   DEPTH   ----   DEPTH  ----   OPT  OPT  OPT         
+EQUIL                                                                        
+         2000.0  195.0   2070    0.0    500.0  0.0    1    0    0    /
+--
+--       DEPTH    PSAT                                                 
+--                PRESS                                                  
+--       ------   ------                                                               
+PBVD            
+         2000.0   75.0  
+         2150.0   75.0                 / PSAT VS DEPTH EQUIL REGN 01          
+--
+--       RESTART CONTROL BASIC = 4 (ALL=2, YEARLY=4, MONTHLY=5, TSTEP=6)
+--
+RPTRST                                                                           
+         BASIC=2  PBPD                 /        
+--
+--       DEFINE SOLUTION SECTION REPORT OPTIONS
+--
+RPTSOL                                                                           
+         FIP=2    FIPRESV              /
+       
+-- =================================================================================================================================
+-- 
+-- SUMMARY SECTION 
+-- 
+-- =================================================================================================================================
+SUMMARY
+--
+--       ACTIVATE COLUMNAR SUMMARY DATA REPORTING OPTION     
+--
+RUNSUM                                               
```

Comment:
Quick note: If the end goal of this work is to run the model as part of the regression test suite, then I would advise **not** using `RUNSUM`.  We've had a number of regression test failures in the past just because the `.RSM` file did not exactly match the main summary output file.

Reply:
Thanks - I want to add a test comparing `./flow pyaction/PYACTION_NEXT_INSERT_KW.DATA`  and `./flow actionx/ACTIONX_NEXT.DATA` using this script https://github.com/OPM/opm-simulators/blob/master/tests/run-comparison.sh - then I'd have to remove the `RUNSUM` keyword in both files, right?

Reply:
> Thanks - I want to add a test comparing [two models] - then I'd have to remove the `RUNSUM` keyword in both files, right?

Yes, that'd be the way I'd do it myself.